pkgs/applications/misc/cortex/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchgit, python3 }:
+
+stdenv.mkDerivation {
+  name = "cortex";
+
+  src = fetchgit {
+    url = "https://github.com/gglucas/cortex";
+    rev = "e749de6c21aae02386f006fd0401d22b9dcca424";
+    sha256 = "d5d59c5257107344122c701eb370f3740f9957b6b898ac798d797a4f152f614c";
+  };
+
+  buildInputs = [ stdenv python3 ];
+
+  prePatch = ''
+    substituteInPlace cortex --replace "/usr/bin/env python3" "${python3}/bin/python3"
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp cortex $out/bin/
+    chmod +x $out/bin/cortex
+  '';
+
+  meta = {
+    homepage = "http://cortex.glacicle.org";
+    description = "An ncurses reddit browser and monitor";
+    license = stdenv.lib.licenses.mit;
+    maintainers = [ maintainers.matthiasbeyer ];
```

Comment:
`maintainers` is defined in `stdenv.lib`. Therefore you have to use `stdenv.lib.maintainers.matthiasbeyer`. You see often `maintainers.name` because `maintainers` was put into scope using `with stdenv.lib`. See for instance https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/typesetting/halibut/default.nix